features/step_definitions/test_steps.rb

```diff
@@ -227,4 +227,21 @@
 
 Then(/^I print all the text contents in the page using html tag$/) do
   puts browser.html
+end
+
+Then (/^I hover over the text "([^"]*)"$/) do |arg|
+  browser.span(:text => arg).when_present.hover
+end
+
+Then (/^I print the subcategories of class "([^"]*)"$/) do |classname|
+  browser.div(:class => classname).spans(:class => "nav-text").each do |span|
```

Comment:
Okay! My bad, didn't see the difference in the sentences of the two gherkin tests you wrote!